specification/synapse/data-plane/Microsoft.Synapse/stable/2021-04-01/symsAPI.json

```diff
@@ -0,0 +1,2206 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ArtifactsClient",
+    "description": "SyMS API Service Client.",
+    "version": "2021-04-01"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/databases/ExecuteChange": {
+      "post": {
+        "summary": "Batch execution of DDL Payload.",
+        "operationId": "ExecuteChange",
+        "description": "Batch execution of DDL Payload",
+        "x-ms-examples": {
+          "Batch execution of DDL Payload": {
+            "$ref": "./examples/ExecuteChange.json"
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "createArtifactsPayload",
+            "in": "body",
+            "description": "DDLBatch payload containing artifact drafts to be created or deleted or modified",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DDLBatch"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/UserAgentParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully completed the management operation.",
+            "schema": {
+              "$ref": "#/definitions/SyMSAPIDDLResponses"
+            }
+          },
+          "default": {
+            "description": "SyMS error response.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
```

Comment:
Why not refer the common type?

```
"$ref": "../../../../common/v1/types.json#/definitions/CloudError"
```

Reply:
Makes sense. Since this PR is merged - I have it addressed this in a separate PR - [PR: 24469](https://github.com/Azure/azure-rest-api-specs/pull/24469). Please take a look and approve if all looks good.